imagemodal/templates/view.html

```diff
@@ -28,5 +28,5 @@ <h2>{display_name}</h2>
             </button>
         </div>
     </div>
-    <p id="xblock_imagemodal__description__{xblock_id}" class="sr">{description}</p>
+    <p id="xblock_imagemodal__description__{{xblock_id}}" class="sr">{{description}}</p>
```

Comment:
This template had no conditional logic (just variable interpolation)
so conversion was as easy as doubling-up the curly braces :)